lib/ansible/modules/dnf5.py

```diff
@@ -127,17 +127,19 @@
     type: bool
   enable_plugin:
     description:
-      - This is currently a no-op as dnf5 itself does not implement this feature.
       - I(Plugin) name to enable for the install/update operation.
         The enabled plugin will not persist beyond the transaction.
+      - O(disable_plugin) takes precedence in case a plugin is listed in both O(enable_plugin) and O(disable_plugin)
+      - Requires python3-libdnf5 5.2.0.0+
```

Comment:
```suggestion
      - O(disable_plugin) takes a higher precedence in case a plugin is listed in both O(enable_plugin) and O(disable_plugin).
      - Requires python3-libdnf5 5.2.0.0+.
```